WalletWasabi/Services/UpdateManager.cs

```diff
@@ -125,8 +130,8 @@ private async Task<(string filePath, Version newVersion)> GetInstallerAsync(Vers
 			{
 				EnsureToRemoveCorruptedFiles();
 
-				// This should also be done using Tor.
-				// TODO: https://github.com/zkSNACKs/WalletWasabi/issues/8800
+				// Wait a moment to avoid dossing GitHub API.
+				await Task.Delay(2000, cancellationToken).ConfigureAwait(false);
```

Comment:
nit: Can you move this closer to the `await HttpClient.SendAsync` ?

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/12895#discussion_r1577591888
It turned out not to be needed.